nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-api/src/main/java/org/nuxeo/ecm/directory/BaseSession.java

```diff
@@ -443,6 +443,20 @@ public void deleteEntry(String id) throws DirectoryException {
         getDirectory().invalidateCaches();
     }
 
+    /**
+     * @param results the query results without limit and offet
+     * @param limit maximum number of results ignored if less than 1
+     * @param offset number of rows skipped before starting, will be 0 if less than 0.
+     * @return the result with applied liimit and offset
+     * @since 10.1
+     * @see {@link Session#query(Map, Set, Map, boolean, int, int)}
```

Comment:
Add a small one-line sentence at the top of the Javadoc to explain the method.
`liimit` -> `limit`
No need for `{@link }` in a `@see` Javadoc tag.